lzy/channel-manager/src/main/resources/db/channel-manager/migrations/V1_0__Init.sql

```diff
@@ -33,3 +33,73 @@ CREATE TABLE channel_operation
     failed      BOOLEAN                NOT NULL DEFAULT FALSE,
     fail_reason TEXT                   NULL
 );
+
+CREATE TYPE channel_life_status_type AS ENUM ('ALIVE', 'DESTROYING');
+
+CREATE TABLE channels
+(
+    channel_id    TEXT                     NOT NULL,
+    execution_id  TEXT                     NOT NULL,
+    workflow_name TEXT                     NOT NULL,
+    user_id       TEXT                     NOT NULL,
+    channel_name  TEXT                     NOT NULL,
+    channel_spec  TEXT                     NOT NULL,
+
+    life_status   channel_life_status_type NOT NULL,
+    created_at    TIMESTAMP                NOT NULL,
+    updated_at    TIMESTAMP                NOT NULL,
+
+    CONSTRAINT channels_pkey PRIMARY KEY (channel_id)
+);
+
+CREATE UNIQUE INDEX channels_execution_id_channel_name_idx
+    ON channels (execution_id, channel_name);
+
+CREATE TYPE endpoint_life_status_type AS ENUM ('BINDING', 'BOUND', 'UNBINDING');
+
+CREATE TABLE endpoints
+(
+    slot_uri    TEXT                      NOT NULL,
+    "slot_name" TEXT                      NOT NULL,
```

Comment:
why do you quote this column name?

Reply:
It's highlighted with black bold font like a keyword without quotes.